### DISCUSSION TITLE
Allow textInput label seperate to allow to placed on right

### DISCUSSION THREAD

**Discussion by keshavuchiha** [mantinedev/mantine#3537] [Feature requests]
![image](https://user-images.githubusercontent.com/94871950/218158952-9ed372e7-724a-466f-8bc2-b9910cb7078a.png)
By default Mantine only displays on top.
So it should be allowed to change position of label

**Comment by rtivital** [comment]
You can render custom label with `Input.Label` component or customize component styles with Styles API
